// import { catchAsyncError } from "../utils/catchAsyncError";

// const register = catchAsyncError(async (req, res, next) => {});

// const login = catchAsyncError(async (req, res, next) => {});

// export {
//     register,
//     login
// }